docs/theming.rst

```diff
@@ -184,8 +196,16 @@ so ``post.tmpl`` only define the content, and the layout is inherited from ``bas
 
 Another concept is theme inheritance. You do not need to duplicate all the
 default templates in your theme — you can just override the ones you want
-changed, and the rest will come from the parent theme. (Every theme needs a
-parent.)
+changed, and the rest will come from the parent theme.  If your theme does not
```

Comment:
Again, unsupported and should not be documented.

Reply:
What do you think about the compromise "we document that this is unsupported", which is what I'm trying to do here?

Reply:
I’m still not a fan of documenting it at all, but I guess it can stay as-is.